### PR TITLE
Tile merger

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |
| Chore            | ✔/✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -42,5 +42,6 @@
       "jobType": "rasterTilesExporter",
       "taskType": "rasterTilesExporter"
     }
-  }
+  },
+  "tilesProvider": "S3"
```
Comment (asafMasa) [inline config/default.json:46]: add to HELM 
Open an issue to get it from umbrella

File: src/clients/jobManagerWrapper.ts [hunk 2]
```diff
@@ -35,36 +35,32 @@ export class JobManagerWrapper extends JobManagerClient {
...
     const createJobRequest: CreateJobBody = {
-      resourceId: resourceId,
-      version: version,
+      resourceId: data.cswProductId,
+      version: data.version,
       type: this.tilesJobType,
       expirationDate,
       parameters: {
-        ...data,
+        sanitizedBbox: data.sanitizedBbox,
```
Comment (asafMasa) [inline src/clients/jobManagerWrapper.ts:47]: job parameters needs to save more data, callbacks target array etc. we need them for job finish

File: src/createPackage/models/createPackageManager.ts [hunk 3]
```diff
@@ -65,10 +109,29 @@ export class CreatePackageManager {
     return duplicationExist;
   }
 
+  private getSeparator(): string {
```
Comment (asafMasa) [inline src/createPackage/models/createPackageManager.ts:112]: ?

File: src/createPackage/models/createPackageManager.ts [hunk 4]
```diff
@@ -12,50 +16,90 @@ import {
...
+    const duplicationExist = await this.checkForDuplicate(dupParams, callbacks);
     if (!duplicationExist) {
+      const batches: ITileRange[] = [];
+      for (let i = 0; i <= zoomLevel; i++) {
+        batches.push(bboxToTileRange(sanitizedBbox, i));
+      }
+      const separator = this.getSeparator();
+      const sources: IMapSource[] = [
+        {
+          path: this.generatePackageName(dbId, zoomLevel, sanitizedBbox), //gpkg path
```
Comment (asafMasa) [inline src/createPackage/models/createPackageManager.ts:76]: ?

File: src/clients/jobManagerWrapper.ts (hunk 2 above)
Comment (cl-shaharshavit-rnd) [reply src/clients/jobManagerWrapper.ts:47]: callbacks is the callbacks target array,
and the layer identifiers are part of the jobs table and therefore aren't needed in the parameters json

File: src/createPackage/models/createPackageManager.ts (hunk 4 above)
Comment (cl-shaharshavit-rnd) [reply src/createPackage/models/createPackageManager.ts:76]: geo package file name is required for the tile merger API.
it was previously generated in the imploder

File: src/createPackage/models/createPackageManager.ts (hunk 3 above)
Comment (cl-shaharshavit-rnd) [reply src/createPackage/models/createPackageManager.ts:112]: required for windows computability

**Review by asafMasa — APPROVED** [review]

**Pull request merged**